ee/fleetd-chrome/src/tables/network_interfaces.ts

```diff
@@ -5,6 +5,10 @@ export default class TableNetworkInterfaces extends Table {
   columns = ["mac", "ipv4", "ipv6"];
 
   async generate() {
+    if (!chrome.enterprise) {
+      return { data: [] };
```

Comment:
Should we add an error/warning?

Reply:
Sure. This should only be seen in dev mode.